provider/datagen/src/baked_exporter.rs

```diff
@@ -29,11 +31,17 @@
 //! # let _ = std::fs::remove_dir_all(&demo_path);
 //! ```
 //!
-//! The resulting module structure can now be used like this:
+//! There are two ways to use baked data: you can build custom data providers for use with
+//! [`_unstable` constructors](icu_provider::constructors), or you can use it with the
+//! `compiled_data` Cargo feature and constructors.
+//!
+//! ## Custom `DataProvider`
+//!
+//! This allows you to use baked data in custom data pipelines, such as including some baked
+//! data and lazily loading more data from the network.
 //!
 //! ```
```

Comment:
Nit: This example should reference `icu4x_baked_demo`, too

 > Markus: "icu4x_baked_demo" is in the first example but I don't see it referenced in the second example. Be more clear about how to get from the output directory of the exporter to the input directory of the runtime provider